module/support/logging/FileLogHandler/src/FileLogHandler.cpp

```diff
@@ -54,14 +54,14 @@ namespace module::support::logging {
 #ifndef NDEBUG  // We have a cold hearted monstrosity that got built!
```

Comment:
I can't review this properly because it wasn't changed in this PR but just above this on lines 51 and 52 there is still some camel case